locales/locales.php

```diff
@@ -2050,6 +2050,15 @@ public function __construct() {
 		$rhg->slug = 'rhg';
 		$rhg->nplurals = 1;
 		$rhg->plural_expression = '0';
+		
+		$rif = new GP_Locale();
+		$rif->english_name = 'Tarifit';
+		$rif->native_name = 'Tarifiyt';
```

Comment:
```suggestion
		$rif->native_name = 'Tmaziɣt';
```
See https://dbpedia.org/page/Tarifit and https://en.wikipedia.org/wiki/Tarifit